README.md

```diff
@@ -1,44 +1,134 @@
 
-Note that in the doc directory there is BookGA.pdf which is a collection of notes on 
-Geometric Algebra and Calculus based of "Geometric Algebra for Physicists" by Doran and 
-Lasenby and on some papers by Lasenby and Hestenes.
+![PyPI - Status](https://img.shields.io/pypi/status/galgebra.svg) ![GitHub contributors](https://img.shields.io/github/contributors/pygae/galgebra.svg) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/fe7642c639a54d909a36c75db6c2fa49)](https://app.codacy.com/app/utensilcandel/galgebra?utm_source=github.com&utm_medium=referral&utm_content=pygae/galgebra&utm_campaign=Badge_Grade_Settings) [![Codecov](https://img.shields.io/codecov/c/github/pygae/galgebra.svg)](https://codecov.io/gh/pygae/galgebra)
+
+[brombo/galgebra](https://github.com/brombo/galgebra) was originally written by Alan Bromborsky, but is no longer actively maintained.
+
+[pygae/galgebra](https://github.com/pygae/galgebra) is a community fork, maintained by [Pythonic Geometric Algebra Enthusiasts](https://github.com/pygae).
+
+The fork supports Python 3, increases test coverage, set up CI and linters, maintains releases to [PyPI](https://pypi.org/project/galgebra/#history), improves [docs](http://galgebra.readthedocs.io) and has many bug fixes.
+
+Features
+--------------------
+
+### Geometric Algebra
+
+- Arbitrary Vector Basis and Metric
+- Scalar, Vector, Bivector, Multivector, Pseudoscalar, Spinor, Blade
+- Basic Geometic Algebra Operations
+  - Sum Difference
+  - Geometric Product
+  - Outer and Inner Products
+  - Left and Right Contractions
+  - Reverse, Dual, Exponential
+  - Commutator
+  - Projection, Reflection, Rotation
+  - Reciprocal Frames
+- Inspecting Base/Blade Representation
+- Symbolic Manipulations
+  - `expand`, `factor`, `simplify`, `subs`, `trigsimp` etc.
+
+Overloaded Python operators for basic GA operations:
+
+![](https://raw.githubusercontent.com/pygae/galgebra/nbsphinx/doc/images/basic_op.svg?sanitize=true)
+
+### Geometric Calculus
 
-Installing Galgebra
+- Geometric Derivative
+- Submanifolds
+- Linear Transformations
+- Differential Operators
+
+The various derivatives of a multivector function is accomplished by multiplying the gradient operator vector with the function:
+
+![](https://raw.githubusercontent.com/pygae/galgebra/nbsphinx/doc/images/grad.svg?sanitize=true) ![](https://raw.githubusercontent.com/pygae/galgebra/nbsphinx/doc/images/grad_cmp.svg?sanitize=true)
+
+Tip: an example for getting `grad` and `rgrad`:
+
+```python
+(ex, ey, ez, grad, rgrad) = 
+  MV.setup('e*x|y|z', metric='[1,1,1]', coords=symbols('x y z'))
+```
+
+### Printing
+
+- Enhanced Console Printing
+- Latex Printing
+  - out-of-the-box support for Jupyter Notebook
+  - PDF generation and croping support if you have `pdflatex`/`pdfcrop` installed
+
+Getting Started
 ---------------------
 
-### Dependencies
+After installing GAlgebra (see Section [Installing GAlgebra](#installing-galgebra) below), in a Jupyter Notebook:
```

Comment:
I'd have gone with "Migration Guide section" instead, which IMO feels more natural in English.